platform/linux-generic/include/odp/api/plat/timer_types.h

```diff
@@ -24,9 +24,9 @@ extern "C" {
  *  @{
  **/
 
-struct odp_timer_pool_s; /**< Forward declaration */
+struct _timer_pool_s; /**< Forward declaration */
```

Comment:
The only _ prefix we've been using is _odp, so I'd either change this to use _odp or else just remove odp_ altogether than leave this as timer_pool_s, etc.

Reply:
I recommend we modify the Coding Style to say ```odp_``` prefix is reserved for the API, and internal data structures should just drop the ```odp_``` prefix.

API data type: ```odp_timer_pool_t```
Internal data type: ```timer_pool_t```